src/store/models/taro.ts

```diff
@@ -1,9 +1,13 @@
 import { action, Action, thunk, Thunk, thunkOn, ThunkOn } from 'easy-peasy';
-import { Status, TaroNode } from 'shared/types';
+import * as TARO from 'shared/tarodTypes';
+import { LightningNode, Status, TarodNode, TaroNode } from 'shared/types';
 import * as PTARO from 'lib/taro/types';
 import { StoreInjections } from 'types';
+import { BLOCKS_TIL_CONFIRMED } from 'utils/constants';
 import { RootModel } from './';
 
+export const TARO_MIN_LND_BALANCE = 10000;
```

Comment:
Add a comment for this const so it's clear what this represents.

Reply:
Done.